src/item.h

```diff
@@ -606,7 +606,7 @@ class item : public visitable
          * If `practical` is false, returns pre-Cataclysm market value,
          * otherwise returns approximate post-cataclysm value.
          */
-        int price_no_contents( bool practical, std::optional<int> price_override = std::nullopt ) const;
```

Comment:
You shouldn't remove the `= std::nullopt`, that's the default value and the reason you had to add `std::nullopt` to the function calls.